examples/secureFields/CardNumber/0-Simple.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import initMercadoPago from '../../../src/mercadoPago/initMercadoPago';
+import CardNumber from '../../../src/secureFields/cardNumber';
+
+initMercadoPago('TEST-f4563544-ce69-40c3-b88e-6e7d1bd93a83', { locale: 'pt-BR' });
+
+const App = () => {
+  return (
+    <CardNumber 
+      placeholder='Card Number'
+      style={{ fontFamily: "sans-serif" }}
+      onReady={() => console.log('Card number ready!!')}
```

Comment:
Acho que vale a pena dizer que o event é recebido como parâmetro, e além disso, acho que seria legal colocar o evento de onError também, que o mais usado de todo os eventos. ai no caso ficaria algo tipo 
`onError={(error) => console.log('Card number error!!', error)}`

```suggestion
      onReady={(event) => console.log('Card number ready!!', event)}
```

Não vou repetir o comentário mas vale para todos os campos